WEB_SERVICE/WEB/_HIGH/4XX_BYPASS/description.md

```diff
@@ -0,0 +1,32 @@
+
+# 4XX-Bypass
+
+The 4xx-bypass exploits refer to HTTP request fuzzing techniques that allow attackers to bypass server-side restrictions, accessing unauthorized resources or performing malicious actions on a web application. This can lead to unauthorized data access, privilege escalation, and other security risks.
+
+=== "Python"
+  ```python
+    import requests
+
+    response = requests.get("http://www.some-url.com/unauthorized_path")
+
+    '''if we have some unauthorized path that gets us a 403 code,
+    we can try something like adding a query parameter to see if we can
+    trick the server by exploiting some mistake.'''
+
+    response = requests.get("http://www.some-url.com/unauthorized_path?debug=true")
+
+    '''Might get us the resource we want'''
+  ```
+
+=== "Python"
+  ```python
```

Comment:
Both Python, why?

Reply:
I removed the doubled code block ticks

Reply:
I added other language for the backend/server part